packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -87,17 +84,18 @@ const WRAPPER_PATH = 'webxdc-wrapper.45870014933640136498.html'
 
 const BOUNDS_UI_CONFIG_PREFIX = 'ui.desktop.webxdcBounds'
 
-const DEFAULT_BOUNDS_WEBXDC = {
-  width: 375,
-  height: 667,
-}
-
-const DEFAULT_BOUNDS_MAP = {
-  width: 1000,
-  height: 800,
-}
+type Size = { width: number; height: number }
 
 export default class DCWebxdc {
+  private DEFAULT_SIZE_WEBXDC: Size = defaultSize({
```

Comment:
Has to be computed here. On file scope isn't possible since we don't know about the primary display yet.

Reply:
I wouldn't calculate this on delta chat application start, the user might connect an external monitor during the session and change their primary display to it. I think the calculation should be done always, also after retrieving the last used size - that might be from a different session where the screen size has been different.

Reply:
I'll tweak it.